### PR TITLE
Fix double bubble inputs

### PR DESCRIPTION
**Pull request by ajnonaka**
remove &probin

### CODE REVIEW

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 1]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
-
-  species_b_name = "B"
-  species_b_gamma = 1.4d0
-
-  species_c_name = "C"
-  species_c_gamma = 1.666d0
-
-/
+# override the default values of the probin namelist values here
+dens_base = 1.65e-3
```
Comment (zingale) [inline Exec/test_problems/double_bubble/inputs_2d_C:97]: shouldn't this be `problem.dens_base`?

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 2]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
+# Note that some of the parameters in this
+# namelist are specific to the default EOS,
+# network, and/or integrator used in the
+# makefile. If you try a different set of
+# microphysics routines be sure to check that
+# the parameters in here are consistent, as
+# Fortran does not like seeing unknown variables
+# in the namelist.
+
+eos.const_conductivity = 1.0d0
```
Comment (zingale) [inline Exec/test_problems/double_bubble/inputs_2d_C:117]: this should be `conductivity.const_conductivity`, also get rid of the `d` -- C++ doesn't like that.

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 3]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
+eos.species_a_name = "A"
+eos.species_a_gamma = 1.666d0
+
```
Comment (zingale) [inline Exec/test_problems/double_bubble/inputs_2d_C:120]: get rid of the `d` in all of these

File: Exec/test_problems/double_bubble/inputs_2d_regression [hunk 4]
```diff
@@ -99,36 +99,31 @@ amrex.regtest_reduction = 1
...
+# Note that some of the parameters in this
+# namelist are specific to the default EOS,
+# network, and/or integrator used in the
+# makefile. If you try a different set of
+# microphysics routines be sure to check that
+# the parameters in here are consistent, as
+# Fortran does not like seeing unknown variables
+# in the namelist.
+
+eos.const_conductivity = 1.0e0
```
Comment (zingale) [inline Exec/test_problems/double_bubble/inputs_2d_regression:120]: `conductivity.const_conductivity`

File: Exec/test_problems/double_bubble/inputs_2d_C (hunk 1 above)
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_C:97]:
```suggestion
problem.dens_base = 1.65e-3
```

Comment (ajnonaka) [reply Exec/test_problems/double_bubble/inputs_2d_C:97]: ack, missed a bunch of things in this one

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 5]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
-  species_b_gamma = 1.4d0
-
-  species_c_name = "C"
-  species_c_gamma = 1.666d0
-
-/
+# override the default values of the probin namelist values here
+dens_base = 1.65e-3
+problem.pert_factor = 8.1e-3
+dens_base = 1.65e-3
```
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_C:99]:
```suggestion
problem.dens_base = 1.65e-3
```

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 6]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
+problem.pres_base = 1.65e6
+dens_base = 1.65e-3
```
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_C:101]:
```suggestion
problem.dens_base = 1.65e-3
```

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 7]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
+
+eos.const_conductivity = 1.0d0
+eos.species_a_name = "A"
+eos.species_a_gamma = 1.666d0
+
+eos.species_b_name = "B"
+eos.species_b_gamma = 1.4d0
+
+eos.species_c_name = "C"
+eos.species_c_gamma = 1.666d0
```
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_C:125]:
```suggestion
conductivity.const_conductivity = 1.0e0
eos.species_a_name = "A"
eos.species_a_gamma = 1.666e0

eos.species_b_name = "B"
eos.species_b_gamma = 1.4e0

eos.species_c_name = "C"
eos.species_c_gamma = 1.666e0
```

File: Exec/test_problems/double_bubble/inputs_2d_regression (hunk 4 above)
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_regression:120]:
```suggestion
conductivity.const_conductivity = 1.0e0
```

Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_regression]:
```suggestion
conductivity.const_conductivity = 1.0e0
```

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 8]
```diff
@@ -93,36 +93,33 @@ maestro.small_temp = 1.e-5
...
-  species_c_gamma = 1.666d0
-
-/
+# override the default values of the probin namelist values here
+problem.dens_base = 1.65e-3
+problem.pert_factor = 8.1e-3
+problem.dens_base = 1.65e-3
+problem.pres_base = 1.65e6
+problem.dens_base = 1.65e-3
+problem.dens_base = 1.65e-3
```
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_C:102]:
```suggestion
problem.dens_base = 1.65e-3
problem.pert_factor = 8.1e-3
problem.pres_base = 1.65e6
```

File: Exec/test_problems/double_bubble/inputs_2d_C [hunk 9]
```diff
@@ -93,36 +93,30 @@ maestro.small_temp = 1.e-5
...
-  species_b_gamma = 1.4d0
-
-  species_c_name = "C"
-  species_c_gamma = 1.666d0
-
-/
+# override the default values of the probin namelist values here
+problem.dens_base = 1.65e-3
+problem.pert_factor = 8.1e-3
+problem.pres_base = 1.65e6
```
Comment (ajnonaka) [inline Exec/test_problems/double_bubble/inputs_2d_C:99]:
```suggestion
problem.pert_factor = 8.1e-3
problem.pres_base = 1.65e6
problem.dens_base = 1.65e-3
```

**Review by zingale — APPROVED** [review]

**Review by doreenfan — APPROVED** [review]

**Pull request merged**